src/jaegertracing/Reference.h

```diff
@@ -30,6 +30,9 @@ namespace thrift {
 class SpanRef;
 }
 
+// An extension of enum opentracing::SpanReferenceType, for a new Span. Only to copy traceID and spanID
+const static int SpanReferenceType_JaegerSpecific_SelfRef = 99;
```

Comment:
does this need to be in `.h`? It sounds like an implementation detail.

Reply:
Agree, will move to a .cpp or at least inside the Reference class (it has no .cpp)

Reply:
Done